rpc/errors.go

```diff
@@ -0,0 +1,15 @@
+package rpc
```

Comment:
Looks like `rpc` needs to grow a top-level package just for this error that's reused in two places. OK, but I liked the plain organization in the directory.

Reply:
Could you please clarify if you're suggesting creating a separate top-level RPC package specifically for centralizing all RPC-related errors?

Reply:
> Could you please clarify if you're suggesting creating a separate top-level RPC package specifically for centralizing all RPC-related errors?

It's more that I'm stating what you have already done in this PR.
There is currently no other purpose for the top-level `rpc` package.

Reply:
no action item here, will mark as resolved